packages/metrics/scripts/reportGenerator.ts

```diff
@@ -0,0 +1,859 @@
+          informantRelationship: ''
+        }
+
+        //Address
+        deathRow.healthCenter = fullComposition.healthCenter ?? ''
+        deathRow.officeLocation = fullComposition.officeLocation ?? ''
+        deathRow.deathDistrict = fullComposition.deceased?.district ?? ''
+        deathRow.deathState = fullComposition.deceased?.state ?? ''
+        deathRow.deathCity = fullComposition.deceased?.city ?? ''
+
+        //Deceased details
+        deathRow.deceasedGen = fullComposition.deceased?.gender ?? ''
+        deathRow.deceasedDOB = fullComposition.deceased?.birthDate ?? ''
+        deathRow.deceasedMaritalStatus =
+          fullComposition.deceased?.maritalStatus ?? ''
+        deathRow.deceasedDate = fullComposition.deceased?.deceasedDate ?? ''
+
+        //Informant details
+        deathRow.informantDOB = fullComposition.informant?.birthDate ?? ''
+        deathRow.informantMaritalStatus =
+          fullComposition.informant?.maritalStatus ?? ''
+        deathRow.informantOccupation =
+          fullComposition.informant?.occupation ?? ''
+        deathRow.informantCity = fullComposition.informant?.city ?? ''
+        deathRow.informantDistrict = fullComposition.informant?.district ?? ''
+        deathRow.informantState = fullComposition.informant?.state ?? ''
+        deathRow.informantRelationship =
+          fullComposition.informant?.relationship ?? ''
+
+        //Observations
+        deathRow.uncertifiedMannerOfDeath =
+          fullComposition.observations?.uncertifiedMannerOfDeath ?? ''
+        deathRow.verbalAutopsyDescription =
+          fullComposition.observations?.verbalAutopsyDescription ?? ''
+        deathRow.causeOfDeathMethod =
+          fullComposition.observations?.causeOfDeathMethod ?? ''
+        deathRow.causeOfDeathEstablished = fullComposition.observations
+          ?.causeOfDeathEstablished
+          ? 'Yes'
+          : 'No'
+        deathRow.causeOfDeath = fullComposition.observations?.causeOfDeath ?? ''
+        deathRow.numMaleDependentsOnDeceased =
+          fullComposition.observations?.numMaleDependentsOnDeceased ?? ''
+        deathRow.numFemaleDependentsOnDeceased =
+          fullComposition.observations?.numFemaleDependentsOnDeceased ?? ''
+
+        // tslint:disable-next-line
+        console.log('Writing CSV row for death declaration...')
+        await deathCSVWriter.writeRecords([deathRow])
+      }
+    }
+    // tslint:disable-next-line
+    console.log(
+      'Successfully generated CSV report for birth and death declarations.'
+    )
+  } catch (error) {
+    // tslint:disable-next-line
+    console.log('Sorry. Something went wrong!', error)
+  }
+}
+
+const startScript = async () => {
+  await connect()
+  const compositionsCursor = await getCompositionCursor()
+  const locations: fhir.Location[] = await getCollectionDocuments(
+    COLLECTION_NAMES.LOCATION,
+    []
+  )
+  await makeCompositionAndExportCSVReport(compositionsCursor, locations)
```

Comment:
@sadmananik should the Mongo connection from this script be closed at this point?

Reply:
Yes. thank you.